build.gradle

```diff
@@ -22,8 +22,8 @@ apply plugin: 'signing'
 apply plugin: 'pl.allegro.tech.build.axion-release'
 apply from: 'gradle/dist.gradle'
 
-sourceCompatibility = 1.7
-targetCompatibility = 1.7
+sourceCompatibility = 1.8
```

Comment:
I think we should be able to use the jsr-310 backport library without explicitly required java 8 as minimum